test/utils/addProxyIntegrationHeaders.test.ts

```diff
@@ -40,3 +40,43 @@ describe('addProxyIntegrationHeaders', () => {
     expect(headers.get('x-custom-header')).toBe('custom-value')
   })
 })
+
+describe('getIPFromHeaders', () => {
+  const ipv6 = '2001:67c:198c:906:3b::3c2'
+  const ipv4 = '19.117.63.126'
+
+  it('returns CF-Connecting-IP when only CF-Connecting-IP is set', () => {
+    const headers = new Headers()
+    headers.set('CF-Connecting-IP', ipv4)
+
+    expect(getIPFromHeaders(headers)).toEqual(ipv4)
+  })
+
+  it('returns CF-Connecting-IP when Cf-Pseudo-IPv4 is present but different', () => {
+    const headers = new Headers()
+    headers.set('CF-Connecting-IP', ipv6)
+    headers.set('Cf-Pseudo-IPv4', ipv4)
+
+    expect(getIPFromHeaders(headers)).toEqual(ipv6)
+  })
+
+  it('returns CF-Connecting-IPv6 when Cf-Pseudo-IPv4 matches CF-Connecting-IP', () => {
```

Comment:
I thought we would be adding test cases for `addProxyIntegrationHeaders`, this approach works, too. But I strongly insist to add at least one more test case for `addProxyIntegrationHeaders`, which would be this exact case, 'returns CF-Connecting-IPv6 when Cf-Pseudo-IPv4 matches CF-Connecting-IP'.